src/ploneorg/addonlisting/browser/configure.zcml

```diff
@@ -43,6 +43,15 @@
       template="templates/addon_base_view.pt"
       permission="zope2.View"
       />
+
+  <browser:page
+      name="AddOnFolderView"
+      for="*"
+      layer="zope.interface.Interface"
```

Comment:
this looks wrong

name should be a simple name: simple_view
for should be ploneorg.addonlisting.interfaces.IAddOnFolder
layer should not be necessary or ploneorg.addonlisting.interfaces.IPloneorgAddonlistingLayer
 the the wrong for and layer:
for should be ploneorg.addonlisting.interfaces.IAddOnFolder